### DISCUSSION TITLE
How to search articles in rust magazine?

### DISCUSSION THREAD

**Discussion by iamazy** [RustMagazine/rustmagazine#62] [Q&A] [answered]
Hi, Thanks for you work first. 

[rustmagazine](https://github.com/RustMagazine/rustmagazine) is a great repo for us to learn rust better. but I didn't find place to search articles in [rustmagazine](https://github.com/RustMagazine/rustmagazine) website and I had to scroll the page. Is there any plan to enable this feature?

**Comment by Folyd** [comment, marked as answer]
We have the plan to support this, see https://github.com/zineland/zine/issues/97. Also, welcome contribute it zine.
